### PR TITLE
Double the number of retries for renaming the folder names

### PR DESCRIPTION
**Pull request by rakeshvanga**
Using the existing retries and doubling that number instead of adding a new setting for adding a retry value for just the renaming part.

### CODE REVIEW

File: src/FileDownloader.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ export default class FileDownloader implements IFileDownloader {
                 return Uri.file(fileDownloadPath);
             };
 
-            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries, retryDelayInMs);
+            return RetryUtility.exponentialRetryAsync(renameDownloadedFileAsyncFn, retries * 2, retryDelayInMs);
```
Comment (daniv-msft) [inline src/FileDownloader.ts:126]: Could you please clarify why we are not providing the right `retries` number directly as a setting? It seems strange to take a `retries` parameter but not to respect it. Even if we're using this `retries` value elsewhere, it seems fine to use the same value in both places, or worst case to create two different `retries` variables for each case if separating them is really needed.

Comment (daniv-msft) [inline src/FileDownloader.ts:126]: If I didn't make a mistake in my calculations, if we make an exponential retry (doubling the delay between each request) with 10 retries and an original delay of 100 ms, in the end the user will wait ~2 minutes.
As of today, all retries included we make the user wait ~3 seconds.
That's a big gap, and I expect that if we were just increasing the total delay to maybe be 12s (7 retries) or even 24 seconds (8 retries), we would already see almost all of the improvement without letting the users wait for nothing if the files are really locked.
From what you've seen, do you expect waiting 2 minutes to make a significant difference compared to waiting 24 seconds to rename the file?

Comment (rakeshvanga) [reply src/FileDownloader.ts:126]: Sure, I would make a change in our code itself to increase the `retires`.

Comment (rakeshvanga) [reply src/FileDownloader.ts:126]: Sure, I would just try with 8 retires. I was trying to provide more retires to give ample time to read the folder.

**Pull request closed without merging**